src/plugins/recursion/README.md

```diff
@@ -0,0 +1,64 @@
+
+
+## Introduction
+
+I took [Relax NG](http://relaxng.org/spec-20011203.html#define-ref)'s `ref` and `define` part and tried to "copy" 
+their approach for elektra's specification.
+Until now I haven't really gotten a sensible idea for that though. Maybe we could talk on the next meeting about this.
+
+Also I have no idea what your `struct` plugin actually does. There is no example, just some code snippet from 
```

Comment:
Thank you for the feedback. The struct plugin was written at a time when no specification existed, so everything was solved via contracts.

The fstab plugin might help you to understand. Important to know is that plugins can read each other's configs/needs.

The idea is that you either extend the struct plugin or you write a new plugin that replaces the struct plugin.